docs/control_center_docs/control_center_start_guide.rst

```diff
@@ -0,0 +1,737 @@
+
+************************************
+Downloading and Installing
+************************************
+Downloads for all operating systems can be found on the Control Center's `GitHub <https://github.com/iq-motion-control/iq-control-center/releases>`_.
+
+Windows
+============
+1. Navigate to the most recent release on Control Center's `GitHub <https://github.com/iq-motion-control/iq-control-center/releases>`_
+2. Click on the Windows installer to download it
+
+.. image:: ../_static/control_center_pics/windows/windows_install.png
+    :scale: 50%
+
+3. Unzip the installer, and inside of the unzipped folder, you will see
+
+.. image:: ../_static/control_center_pics/windows/win_unzipped.png
+
+4. Run the application
+5. You will see the IQ Control Center Setup window appear
+
+.. image:: ../_static/control_center_pics/windows/win_installer_pg1.png
+
+6. Complete the installation as instructed by the installer wizard
+7. If successful, you will see the following after selecting Show Details
+
+.. image:: ../_static/control_center_pics/windows/windows_installer_done.png
+
+8. Click Next, then Finish
+9. Now, if you hit the Windows key, and search for IQ Control Center, you should see the Application listed. Click it to start the application
+
+.. image:: ../_static/control_center_pics/windows/windows_key_search.png
+    :scale: 75%
+
+10. Alternatively, navigate to the location where you installed the application
+11. Open the folder, then open IQ Control Center
+
+.. image:: ../_static/control_center_pics/windows/windows_installed_folder.png
+
+12. Inside of that folder, you will find the IQ Control Center executable. Double click it to start the application
+
+.. image:: ../_static/control_center_pics/windows/win_executable.png
+
+13. After opening IQ Control Center, you should see the following
+
+.. image:: ../_static/control_center_pics/windows/win_control_center_open.png
+    :scale: 50%
+
+Congratulations! You have successfully installed IQ Control Center, and are ready to start communicating with your modules.
+
+Linux (Ubuntu)
+=================
+1. Navigate to the most recent release on Control Center's `GitHub <https://github.com/iq-motion-control/iq-control-center/releases>`_
+2. Click on the Linux installer to download it
+   
+.. image:: ../_static/control_center_pics/linux/linux_installer_github.png
+    :scale: 75%
+
+3. Once downloaded, unzip the folder, and you will see the installer
+
+.. image:: ../_static/control_center_pics/linux/linux_installer_icon.png
+
+4. Double click to start the installer 
+   
+.. note::
+    If your installer does not open, please perform the following steps.
+
+    a. Open a new terminal window
+    b. Navigate to the location storing the Control Center Installer
+
+    .. image:: ../_static/control_center_pics/linux/linux_terminal_location.png
+
+    c. Use ./ to run the installer
+    d. If you see the following error You MUST run `sudo apt-get install libxcb-xinerama0`
```

Comment:
Any idea what this thing is for or why people might need to do this?

Reply:
I think it's a graphics/window manager. Doesn't ship by default on linux machines. 

"provides an interface to the X Window System protocol, which replaces the current Xlib interface"